### PR TITLE
XW-2215 | add Contributing documentation

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/XW-2215

### CODE REVIEW

File: app/templates/overview/contributing.hbs [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+</section>
+
+<section>
+	<h2>How to develop</h2>
+	<p>If you want to contribute to DS follow the steps:</p>
+	<ul>
+		<li>Clone <a href="https://github.com/Wikia/design-system">Design System</a> repository</li>
+		<li>Install dependencies with <code class="format-code">npm run setup</code></li>
+		<li>Run <code class="format-code">npm run dev</code> to build the Reference Page and start file watchers</li>
+		<li>Open <a href="http://localhost:4200">localhost</a> to see the Reference Page</li>
```
Comment (rogatty) [inline app/templates/overview/contributing.hbs]: IMO `<a href="http://localhost:4200">http://localhost:4200</a>` would be better here.

File: app/templates/overview/contributing.hbs [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+		<li>Modify the Design System or Reference Page source files as you see fit, Ember CLI will rebuild the RP app and reload the page</li>
+		<li>If you want to present the results of your work using devbox see "How to deploy Reference Page to devbox" section below</li>
```
Comment (rogatty) [inline app/templates/overview/contributing.hbs]: Why not `<a href="#devbox-deploy">...</a>`?

File: app/templates/overview/contributing.hbs [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+		<li>Create template file in <code class="format-code">app/templates/</code>(file path have to match route)</li>
+	</ul>
+</section>
+
+<section>
+	<h2>How to deploy Reference Page to devbox</h2>
+	<p>
+		You can present your work on Reference Page by deploying it to your devbox. To do so you can use
+		<code class="format-code">deploy-debox.sh</code> script which will make the Reference Page available on
+		<a href="http://dev-developer/design-system/">http://dev-developer/design-system/</a>.
```
Comment (rogatty) [inline app/templates/overview/contributing.hbs]: What's the point of a broken link?

File: app/templates/overview/contributing.hbs [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+			This is the only directory thatyou will get when installing DS using Bower.
+			It shouldn’t be modified manually as it’s built by Ember CLI when using npm run build.
+		</li>
+		<li>
+			<code class="format-code">docs</code> - built and ready to use Reference Page. This is what GitHub Pages display on
+			<a href="http://fandomdesignsystem.com">fandomdesignsystem.com</a> (using version from the master branch).
+		</li>
+		<li>
+			<code class="format-code">docs-dev</code> - development version of Reference Page.
+			It’s built when using npm run dev and is accessible on <a href="http://localhost:4200">localhost</a>.
```
Comment (rogatty) [inline app/templates/overview/contributing.hbs]: `<a href="http://localhost:4200">http://localhost:4200</a>`

File: app/templates/overview/contributing.hbs [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+		<li><code class="format-code">ember-cli-build.js</code> - configuration file for the Ember CLI which is our build pipeline.</li>
+	</ul>
+</section>
+
+<section>
+	<h2>Repository APP (Reference Page) structure</h2>
+	<ul>
+		<li>
+			<code class="format-code">components</code> - Definitions of RP-only components, see
+			<a href="http://emberjs.com/api/classes/Ember.Component.html and https://guides.emberjs.com/v2.8.0/components/defining-a-component/">defining a component</a>
```
Comment (rogatty) [inline app/templates/overview/contributing.hbs]: Broken link.

File: app/templates/overview/contributing.hbs [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+			for more details. You need to use these only if you want to expand RP features.
+			Templates live in templates/components directory and it’s possible that a template will be enough for your needs
+			and you won’t need the JS file. A component template is enough to render it.
+		</li>
+		<li>
+			<code class="format-code">controllers</code> - Definitions of RP controllers, see
+			<a href="http://emberjs.com/api/classes/Ember.Controller.html and https://guides.emberjs.com/v2.8.0/controllers/">controllers</a>
```
Comment (rogatty) [inline app/templates/overview/contributing.hbs]: Broken link.

File: app/templates/overview/contributing.hbs [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+	<h2>Repository structure</h2>
+	<ul>
+		<li>
+			<code class="format-code">app</code> - source files for Ember Reference Page application.
+			You will need to edit them when modifying the RP. See "Repository APP (Reference Page) structure" section below
+		</li>
+		<li><code class="format-code">config</code> - configuration for Reference Page. It’s unlikely that you will need to touch it.</li>
+		<li>
+			<code class="format-code">dist</code> - Design System distributable files.
+			This is the only directory thatyou will get when installing DS using Bower.
```
Comment (weronika) [inline app/templates/overview/contributing.hbs]: missing space

File: app/templates/overview/contributing.hbs [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+			It shouldn’t be modified manually as it’s built by Ember CLI when using npm run build.
```
Comment (weronika) [inline app/templates/overview/contributing.hbs]: <code class="format-code"> - for the sake of consistency

File: app/templates/overview/contributing.hbs (hunk 2 above)
Comment (rogatty) [inline app/templates/overview/contributing.hbs]: Turns out Ember doesn't support anchor links when using `locationType: 'hash'`. We would need to use something like https://github.com/mike-north/ember-anchor and I don't think now is a good moment for that.

**Pull request merged**